### PR TITLE
Introduce qpid::config_cmd as qpid-config wrapper

### PR DESCRIPTION
**Pull request by ekohl**

### CODE REVIEW

File: manifests/bind_event.pp [hunk 1]
```diff
@@ -6,30 +6,26 @@
...
 #
 # $port::                       Port that qpid is listening on
 #
 # $ssl_cert::                   SSL cert to use for qpid-config commands
 define qpid::bind_event(
   $queue,
+  $exchange = 'event',
   $hostname = 'localhost',
-  $port = 5671,
+  $port = undef,
```
Comment (ehelms) [inline manifests/bind_event.pp]: Why not default this to the standard qpid port?

**Comment by ehelms** [comment]
I like this proposed addition and would prefer to merge it then re-factor my change on top of this structure.

File: manifests/bind_event.pp (hunk 1 above)
Comment (ekohl) [inline manifests/bind_event.pp]: Because the port is determined in the wrapper. That means it automatically selects the correct port for SSL and non-SSL.

Comment (ehelms) [inline manifests/bind_event.pp]: Touche!

**Pull request merged**